dev-lisp/cmucl/cmucl-21c-r1.ebuild

```diff
@@ -0,0 +1,107 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+MY_PV=${PV:0:3}
+
+DESCRIPTION="CMU Common Lisp is an implementation of ANSI Common Lisp"
+HOMEPAGE="https://www.cons.org/cmucl/"
+SRC_URI="
+	https://common-lisp.net/project/cmucl/downloads/release/${MY_PV}/cmucl-src-${MY_PV}.tar.bz2
+	https://common-lisp.net/project/cmucl/downloads/release/${MY_PV}/cmucl-${MY_PV}-x86-linux.tar.bz2
+"
+S="${WORKDIR}"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="-* ~x86"
+IUSE="X doc source"
+
+CDEPEND=">=dev-lisp/asdf-2.33-r3:=
+	x11-libs/motif:0"
```

Comment:
I'm curious about this -- I was going to say "should we rename X -> gui per QA policy?" but noticed that motif is unconditionally depended on. Is that right?

Reply:
Flag description is "Build CLX, CLM, or Hemlock".
IIUC Motif is needed unconditionally.